### ISSUE TITLE
380 form of work

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#169]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/3XX.csv

**Comment by laura-ake** [comment]
Hi @pan-zhuo Pan, reviewed your mapping for 380 Form of Item, I made these changes:

Subfield 2 Source of term mapping:  Recording Method:  RDF Custom Datatype of the property statement, using URI for the source VES. code.  Transformation Notes:  See Decision https://github.com/uwlib-cams/MARC2RDA/wiki/Decisions-Index#iik-2-in-3xx-and-in-x3065x-with-indicator-2 

Subfield 3 Materials specified mapping:  RDA Registry URI:  http://rdaregistry.info/Elements/w/P10330 ;  Label:  has note on work ;  Method:  unstructured description; ;  Transformation:  Create a note:  
""Category of work "" -->[value of $a] ""applies only to "" -->[value of $3] ""."" 

Subfield 0  Authority record control number or standard number, mapping to p10004 has category of work seems ok, just added transformation note "record as IRI if string begins with "http", else as identifier.  Do not map if it duplicates $1"

I've moved it to Review in Progress.  If you want to change anything you can, otherwise whack it over to Ready for Transform.





@CECSpecialistI

**Comment by pan-zhuo** [comment]
Thanks @laura-ake for reviewing.

What I've been struggling with is the category of an aggregating work, regardless of whether $3 is present or not.

For example, for a book that is a collection of poems, it doesn't seem right to say the aggregating work has category
"Poetry", when an aggregating work is defined as a plan.

The problem is that you can't tell if something is an aggregate from 380 alone...

**Comment by lake44me** [comment]
@pan-zhuo "doesn't feel right" needs to have something behind it.  I don't feel a prickling that there might be anything wrong with applying a category of work to an aggregating work, and the Toolkit page for that property doesn't say anything against it, nor does the Describing a Work gudance section on aggregates.  If you look at some of the PCC advice in the Toolkit around Category of Work, there's specific advice on applying 380 to Series (which are certainly aggregating works, whether monographic or continuing, I would think).  If the ag work is a collection of poems, then sure, the "plan" is to aggregate poems specifically, so why not?

**Comment by CECSpecialistI** [comment]
I think it's ok for aggregating works to have a category of work that is the value of the 380. I see what Zhuo is saying, and "Is the plan to aggregate poems a poem?" is a good question to ask here. I think it doesn't have to be in order for the form of work to be applied. Unless @GordonDunsire or @AdamSchiff think otherwise I think we're ready to move this one to "ready for transform".

**Comment by AdamSchiff** [comment]
We don’t distinguish in category of work between singular or plural because we use (by PCC policy) controlled vocabulary whenever possible.  So an individual poem gets 380 Poetry $2 lcgft and a collection of poems gets the same thing.  One could record “Poetry collection” if that is found in a VES.  There are items in Wikidata for "poetry collection" (Q12106333 ) and for "poetry anthology" (Q19357149) so we could use one of those identifiers/IRIs in the 380 instead of or in addition to an LCGFT or LCSH term, which is always established in the plural.  (WD describes a poetry anthology as "works from multiple poets chosen by an editor.")

Bottom line: I think 380 is fine for aggregates.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by GordonDunsire** [comment]
This is ok with me, but there are, of course, additional factors that we may need to revisit after we see the transform output. I mentioned on the call this week that it would be interesting to see all the values of "has category of work" that arise.

In the case of an aggregating work, there is an emerging consensus that "aggregating work" should be a category:

ex:aW1 rdawd:P10004 "aggregating work" .

The "plan" for aggregating expressions is not embodied as a manifestation that is a plan (outline, process, algorithm, etc.); it is embodied as the selected set of aggregated expressions arranged in a specific sequence and carried on a manifestation that is an aggregate. The "plan" is the abstract idea of the theme or focus of the aggregate.

Some aspects of the plan are best accommodated in representative expression elements. There is a special RDA interpretation of [representative expression of an aggregating work](https://access.rdatoolkit.org/en-US_ala-39db8725-25e2-3bcf-8a45-7ef37ec38c90/section_ygt_gvg_3lb) that covers, for example, the specific content type of the aggregated expressions:

ex:aW1 rdaw:P10349 rdaco:1020 . // content type is "text"

Other aspects are best accommodated with "subject" elements:

ex:aW1 rdawd:P10261 "Burns, Robert, 1759 1796" . // anthology of poems of Burns
ex:aW1 rdaw:P10256 http://id.loc.gov/authorities/subjects/sh85103707 . // LCSH "Poetry--Collections"

There is no established practice for any of this, and I don't think anyone is yet working out the practical detail. The main angels-pin topic is the boundary between "of" and "about" in the context of aggregates: is a collection of Burns' poems "by" Burns or "about" Burns? I go with the latter, but I can foresee much disagreement. I'm looking forward to a general discussion on transforming aggregates in due course :-)

**Comment by CECSpecialistI** [comment]
@cspayne @lake44me would you take a look at this issue and see if it is ready for transform?

**Comment by lake44me** [comment]
I don't see a problem with it.  Didn't know it had been moved back from "Done".  @cspayne the only thing that might be questionable would be handling of $0 - is there some newer discussion that means we should modify the transform notes?

**Comment by cspayne** [comment]
We made a decision for other 3XX fields, such as 337 and 338, that when a source was provided in $2 we would mint a concept so that the source is retained, instead of simply using a text value. I believe I moved this back to almost done because we need to decide if we want to follow this pattern for 380 and update the mapping if so.

**Comment by GordonDunsire** [comment]
@cspayne and others: I see no reason not the follow the minted concept pattern for field 380.

In fact, I think the pattern should be applied to any tag with similar syntax: a term, a source for the term, an IRI for the term, etc. This pattern is part of the set of transform substructures for bridging M21 encoding semantics with RDA/RDF that Phase 1 has developed; a component of the transform toolkit, as it were :-)

**Comment by cspayne** [comment]
I agree.

**Comment by lake44me** [comment]
Chiming in to agree with @GordonDunsire , if we've minted concepts for similar tags, by all means let's do so.  @cspayne can you update the mapping spreadsheet?  This week my spare time has all been eaten by my job.

**Comment by cspayne** [comment]
> Chiming in to agree with [@GordonDunsire](https://github.com/GordonDunsire) , if we've minted concepts for similar tags, by all means let's do so. [@cspayne](https://github.com/cspayne) can you update the mapping spreadsheet? This week my spare time has all been eaten by my job.

Yes! The mapping has been updated and this issue moved to Awaiting Review.

**Comment by cspayne** [comment]
@CECSpecialistI @sarahruska This mapping should be assigned to someone for review

**Comment by CECSpecialistI** [comment]
I can review this one.

**Comment by CECSpecialistI** [comment]
I didn't make any changes to the mapping in the review, just updated the spreadsheet to indicate the review was done.

**Comment by tmqdeborah** [comment]
Question: Why is a Manifestation element showing up in the Work section in this [test output file](https://github.com/uwlib-cams/MARC2RDA/blob/3c3f037801281d172e5c36bf0edd0f3a2f7caccf/Working%20Documents/transformationCode/test_output/3xx/380Test-RDA.xml#L13-L15
)
       <fake:marcfield>F380 ## $a Open educational resources $2 lcsh $3 Introduction</fake:marcfield>
      <rdaw:P10004 rdf:resource="http://marc2rda.edu/fake/lcsh/con#openeducationalresources"/>
      <rdamd:P30137>Category of work Open educational resources applies to the work's Introduction</rdamd:P30137>

I noticed this because I loaded the file into RIMMF and this data showed up in a Work description.
